connectors/automation-anywhere/element-templates/automation-anywhere-connector.json

```diff
@@ -0,0 +1,692 @@
+      "value": "={\"X-Authorization\":\"asObject:token.asString\"}",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.outputHeaders"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "oneOf": [
+          "userNamePasswordAuth",
+          "userNameApiKeyAuth"
+        ]
+      }
+    },
+    {
+      "label": "Room URL",
+      "group": "configuration",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "controlRoomURL"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "method for request",
+      "type": "Hidden",
+      "value": "post",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "method"
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "createWorkItemModel",
+          "createQueues",
+          "addWorkItemsToTheQueue",
+          "listWorkloadQueues",
+          "listAvailableWorkItemModelId"
+        ]
+      }
+    },
+    {
+      "label": "method for request",
+      "type": "Hidden",
+      "value": "put",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "method"
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "updateWorkItem"
+        ]
+      }
+    },
+    {
+      "label": "Authentication URL",
+      "type": "String",
+      "group": "configuration",
+      "optional": false,
+      "value": "",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.request.url"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "oneOf": [
+          "userNamePasswordAuth",
+          "userNameApiKeyAuth"
+        ]
+      }
+    },
+    {
+      "label": "Work Queue ID",
+      "group": "input",
+      "description": "The queue ID of the item",
+      "type": "String",
+      "optional": false,
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "queueId"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "addWorkItemsToTheQueue",
+          "updateWorkItem"
+        ]
+      }
+    },
+    {
+      "label": "Work Item ID",
+      "group": "input",
+      "description": "The work ID of the item",
+      "type": "String",
+      "optional": false,
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "workItemId"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "updateWorkItem"
+        ]
+      }
+    },
+    {
+      "label": "create work item model",
```

Comment:
Move to description.
I won't repeat this comment again - consider it applicable to all similar hidden fields.